components/automate-ui/src/app/components/resource-dropdown/resource-dropdown.component.spec.ts

```diff
@@ -48,15 +49,119 @@ describe('ResourceDropdownComponent', () => {
     });
 
     it('displays a list of checkbox options', () => {
-      const options = Array.from(fixture.nativeElement.querySelectorAll('chef-checkbox'));
+      const options: HTMLInputElement[] = Array.from(fixture.nativeElement.querySelectorAll('chef-checkbox'));
       expect(options.length).toEqual(component.filteredResources.length);
-      options.forEach((option: HTMLInputElement, index: number) => {
+      options.forEach((option, index) => {
         const { name, checked } = component.filteredResources[index];
         expect(option.textContent).toEqual(name);
         expect(option.checked).toEqual(checked);
       });
     });
 
+    it('allows customizing "none" designator', () => {
```

Comment:
Not actually related to the changes in this PR, but lots of necessary tests added.